src/utils/string.ts

```diff
@@ -1,8 +1,4 @@
-export function trimSuffix(str: string | undefined, suffix: string) {
-  if (!str) {
-    return;
-  }
-
+export function trimSuffix(str: string, suffix: string): string {
```

Comment:
I think this breaks https://github.com/tailscale-dev/vscode-tailscale/blob/6eb760448be2339fd456bf75a21e608c7bc48750/src/tailscale/cli.ts#L442

Or from what I can tell, in Go `Self` might actually be nil but we're not specifying so in the TypeScript type decl: https://github.com/tailscale-dev/vscode-tailscale/blob/6eb760448be2339fd456bf75a21e608c7bc48750/src/types.ts#L52

Reply:
Reverted the change and accounted for it returning undefined.

Reply:
Well, now waiting for it to show up on Github...

Reply:
Ok, closing and re-opening the PR fixed it. Github lost that this PR was associated with the branch somehow as seen in the branch list.